src/test/java/utility/ParserTest.java

```diff
@@ -0,0 +1,31 @@
+package utility;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    
+    @Test
+    public void extractCommandTest() {
```

Comment:
```suggestion
    public void parserExtractCommand_validCommand_success() {
```
`extractCommandTest` is vague in terms of describing the expected input and result the test case should produce. Perhaps `parserExtractCommand_validCommand_success` might be better?